src/Components/v2/Carousel.tsx

```diff
@@ -240,6 +239,9 @@ export class BaseCarousel extends React.Component<
   checkLastItemVisible = () => {
     if (this.flickity && this.flickity.selectedElements) {
       const lastItemVisible = this.flickity.selectedElements.includes(
+        // FIXME: Undocumented API. Is there a way this can be achieved with
+        // something public and commonly available?
+        // @ts-ignore
```

Comment:
@kierangillen - Where did you discover `getLastCell`? It's not in the docs but I've confirmed it is indeed in the source; was it from inspecting the `flickity` instance? Is there a way to use a public API method for this, something in the docs? If not can we open a PR to flickity updating the docs and then update the corresponding type on DefinitelyTyped?